documentation/modules/auxiliary/scanner/snmp/snmp_enumusers.md

```diff
@@ -0,0 +1,33 @@
+## Description
+This module queries a range of hosts via SNMP and gathers a list of usernames on the remote system.
+
```

Comment:
Same as above, did you just install the default snmp on windows, was there other things needed to get this to be vulnerable?